lib/localjob.rb

```diff
@@ -10,7 +10,7 @@ class Localjob
   extend Forwardable
 
   attr_reader :name
-  attr_accessor :queue
+  attr_writer :queue
```

Comment:
The `attr_accessor` was providing a getter and a setter and the method on line 26 overriding the getter causing a warning.